lib/parameters.py

```diff
@@ -114,14 +140,19 @@ def gen_avc_vbr_variants(spec, profiles):
           variant.get("refs", 1), profile
         ]
 
-def gen_avc_vbr_parameters(spec, profiles):
+def gen_avc_vbr_parameters(spec, profiles, strapi=None):
```

Comment:
```suggestion
def gen_avc_vbr_parameters(spec, profiles, strapi=False):
```
Please change all of these.  Thanks.